database/migrations/2021_12_08_113403_create_brands_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBrandsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('brands', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
is it marked as primary key?

Reply:
Yes. The increments method creates an auto-incrementing UNSIGNED INTEGER equivalent column as a primary key.

Reply:
ok